frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -202,15 +206,17 @@ pods:
                             -f ./apache-cassandra-{{CASSANDRA_VERSION}}/conf/cassandra.yaml \
                             -d ${LOCAL_SEEDS} \
                             -p ${CASSANDRA_NATIVE_TRANSPORT_PORT} "$t" ;
+                    result=$?
```

Comment:
- can `set -e` be called at the start of the `cmd`?
- the block's indent is off